### PR TITLE
3 guidelines java

### PR DESCRIPTION
**Pull request by hubertborysowski**

### CODE REVIEW

File: guidelines.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## Programming Style
+
+For programming in Java we use [google style](https://google.github.io/styleguide/javaguide.html)
```
Comment (Mateusz-Dobrzynski) [inline guidelines.md]: Missing capital letter in `Google`

File: guidelines.md
Comment (Mateusz-Dobrzynski) [inline guidelines.md:1]: Put this file in the `doc` directory and change the name to uppercase, i.e. `CONTRIBUTING.md`, as in the project structure guide.

File: guidelines.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+## Programming Style
+
+For programming in Java we use [google style](https://google.github.io/styleguide/javaguide.html) 
+
+## Javadoc
```
Comment (Mateusz-Dobrzynski) [inline guidelines.md]: This section is possibly redundant, as Google Style Guide includes a Javadoc section

**Review by Mateusz-Dobrzynski — CHANGES_REQUESTED** [review]

**Comment by hubertborysowski** [comment]
GUIDELINES.md path has just been changed. All the commented issues are resolved now.

**Review by Mateusz-Dobrzynski — APPROVED** [review]
Change the file name to `CONTRIBUTING.md`

**Pull request merged**